blogposts/examples/how-to-write-blogposts.md

```diff
@@ -0,0 +1,638 @@
+---
+title: 'How to write blogposts'
+date: '2024-02-01'
+tags: ['Unit 214', 'Blog', 'Tutorial']
+author: 'Timo'
+previewText: Blogposts are a cool way of presenting content. In this article you will learn how to easily create your own blogposts using Markdown.
```

Comment:
there are quite a few things here that do not align with the example, is this because of prettier?

Reply:
What do you mean by they do not align with the example?